### PR TITLE
sync with yea-wandb (live mockserver)

### PR DESCRIPTION
**Pull request by raubitsj**
Description
-----------
Update to use the latest yea-wandb

This has been reviewed at:
https://github.com/wandb/yea-wandb/pull/47

Testing
-------
unittests

### CODE REVIEW

File: tests/utils/mock_server.py [hunk 1]
```diff
@@ -344,12 +349,114 @@ def to_dict(self):
...
+
+            # Normal mockserver mode, disable live relay and call next function
+            if not os.environ.get("MOCKSERVER_RELAY"):
+                return func(*args, **kwargs)
+
+            if request.method == "POST":
+                url_path = request.path
+                body = request.get_json()
+
+                url = f"https://api.wandb.ai{url_path}"
```
Comment (kptkin) [inline tests/utils/mock_server.py:373]: i don't know the format of `request.path` is it already has proper separators?

Comment (raubitsj) [reply tests/utils/mock_server.py:373]: yep.  it is the part of the url after the hostname starting with a /

File: tests/utils/mock_server.py [hunk 2]
```diff
@@ -344,12 +349,114 @@ def to_dict(self):
...
+                    time_now = time.time()
+                    if self._inject_count == 0:
+                        self._inject_time = time_now
+                    self._inject_count += 1
+                    if time_now < self._inject_time + 21:
+                        # print("INJECT", self._inject_count, time_now, self._inject_time)
+                        time.sleep(12)
+                        raise HttpException("some error", status_code=500)
+                return data
+            assert False
```
Comment (kptkin) [inline tests/utils/mock_server.py:400]: why do we have this assertion?

Comment (raubitsj) [reply tests/utils/mock_server.py:400]: it isnt an assertion..  it is an injected error.   see the description of the original PR for the backstory: 
https://github.com/wandb/yea-wandb/pull/47

**Review by kptkin — APPROVED** [review]
Approving as it already been reviewed here:
https://github.com/wandb/yea-wandb/pull/47/files

**Pull request merged**